buttonHandler.js

```diff
@@ -49,19 +66,23 @@ ButtonHandler.prototype.isTrue = function(target, eventCount, mainWindow) {
 	this.statsIsTrueInvokedCount++;
 	var result;
 	if (this.optionalIsTrueFunction == null) {
+		if (this.debugLogElementTreeOnIsTrue) {
+		 UIATarget.localTarget().logElementTree();
+	    }
 		var aButton = this.findButton(target);
-        result = aButton.isNotNil() && aButton.validAndVisible();
-    } else {
-	    result = this.optionalIsTrueFunction(target, eventCount, mainWindow);
-    }
-    if (result) {
-	  this.statsIsTrueReturnedTrue++;
-    } else {
-	  this.statsIsTrueReturnedFalse++;
-    };
-    return result;
+		result = this.isValidAndVisible(aButton);
```

Comment:
Here I go again with leading white space nits. :) Can you double check if there's a tab vs spaces thing here? There's more lines changed than need to for this patch.